src/lib/core/overlay/scroll/scroll-dispatcher.ts

```diff
@@ -19,18 +20,15 @@ export class ScrollDispatcher {
   /** Subject for notifying that a registered scrollable reference element has been scrolled. */
   _scrolled: Subject<void> = new Subject<void>();
 
+  /** Keeps track of the global `scroll` and `resize` subscriptions. */
```

Comment:
Add comment about when this should have subscriptions (when there is at least one actively registered scrollable).